src/pages/CollectionSingleViewPage.tsx

```diff
@@ -61,6 +68,7 @@ export const CollectionSingleViewPage = () => {
   const history = useHistory();
   const locationState = useLocation();
   const [tab, setTab] = useState(0);
+  const [mutationType, setMutationType] = useState('aa');
```

Comment:
If we wrap the `FormControl` and the `MutationTableTabContent` in another component, we can move this state out of this far-too-big component into that other component.